content/bn/kubernetes.md

```diff
@@ -0,0 +1,28 @@
+---
+title: কুবারনেটিস
+status: Feedback Appreciated
+category: প্রযুক্তি
+---
+
+## এটা কি
+
+কুবারনেটস (Kubernetes), প্রায়ই K8s হিসাবে সংক্ষিপ্ত হয়, একটি ওপেন সোর্স কন্টেইনার অর্কেস্ট্রেটর যা আধুনিক পরিকাঠামোতে [কন্টেইনারাইজড](/container/) অ্যাপ্লিকেশনের জীবনচক্রকে স্বয়ংক্রিয় করে। এটি একটি ডেটা সেন্টারের অপারেটিং সিস্টেমের মতো, যা অংশ-বিভাগিত সিস্টেম (Distributed System) জুড়ে চলমান অ্যাপ্লিকেশনগুলিকে পরিচালনা করে (ঠিক যেমন আপনার ল্যাপটপের OS আপনার অ্যাপগুলি পরিচালনা করে) ‍।
+
+কুবারনেটস একটি [ক্লাস্টারের](/cluster/) নোড জুড়ে কোনটেনারের সময়সূচী নির্ধারণ করে। এটি বেশ কয়েকটি পরিকাঠামো নির্মাণ, যাকে কখনও কখনও "প্রিমিটিভস" হিসাবে উল্লেখ করা হয়, যেমন একটি অ্যাপের দৃষ্টান্ত, লোড ব্যালেন্সার, পারসিষটেন্ট স্টোরেজ এবং অন্যান্যগুলিকে এমনভাবে একত্রিত করে যাতে সেগুলি অ্যাপ্লিকেশনগুলিতে তৈরি করা যায়।
```

Comment:
`কুবারনেটিস`  should be used . Also you need to change the spelling of this word `কোনটেনারের` . Moreover, can you please create a suitable sentence instead of `এটি বেশ কয়েকটি পরিকাঠামো নির্মাণ, যাকে কখনও কখনও "প্রিমিটিভস" হিসাবে উল্লেখ করা হয়, যেমন একটি অ্যাপের দৃষ্টান্ত, লোড ব্যালেন্সার, পারসিষটেন্ট স্টোরেজ এবং অন্যান্যগুলিকে এমনভাবে একত্রিত করে যাতে সেগুলি অ্যাপ্লিকেশনগুলিতে তৈরি করা যায়।` ?